### ISSUE TITLE
Unet训练问题

### ISSUE THREAD

**Issue by Whitman1984** [alibaba/MNN#2612]
平台(如果交叉编译请再附上交叉编译目标平台):
服务器Linux平台：Ubuntu 20.04.2 LTS

Github版本: 2.7.0
Github Version:
首先官方提供的损失函数MAE，MSE，只支持二维的，像Unet这种训练是不是需要自己定义损失函数？
直接计算损失为auto loss = _ReduceMean(_ReduceSum(_Square(predict - example.second[0]), {1}), {});
仍然在训练时候报错solver->step(loss);时报Segmentation fault (core dumped)或者free(): invalid pointer，无法训练
